src/current/molt/molt-fetch.md

```diff
@@ -355,14 +371,24 @@ Before using this option, the source PostgreSQL or MySQL database **must** be co
 	  GROUP BY source_uuid;
 	~~~
 
-	In the `molt fetch` command, [specify a GTID set](#mysql-replication-flags) using the format `source_uuid:min(interval_start)-max(interval_end)`. For example:
+	In the `molt fetch` command, specify a GTID set using the [`--defaultGTIDSet` replication flag](#mysql-replication-flags) and the format `source_uuid:min(interval_start)-max(interval_end)`. For example:
 
 	{% include_cached copy-clipboard.html %}
 	~~~
 	--mode replication-only 
 	--replicator-flags "--defaultGTIDSet 'b7f9e0fa-2753-1e1f-5d9b-2402ac810003:3-21'"
 	~~~
 
+If replication is interrupted, specify the staging schema with the [`--stagingSchema` replication flag](#replication-flags). MOLT Fetch outputs the schema name as `staging database name: {schema_name}` after the initial run of `--mode replication-only`.
```

Comment:
What do you think about changing it to something similar to: "the initial run of either \`--mode replication-only\` or \`--mode data-load-and-replication\`", or "the initial run of replication." Technically, it could be either mode, but do we need exhaustive detail here?

Reply:
Amended and ended up turning this into its own section.